common/bin/oo-diagnostics

```diff
@@ -1204,6 +1204,29 @@ class OODiag
 
   end
 
+  def test_auth_keys_salt_failure
+    if Rails.env.development?
+      msg_log = `grep \'401 Unauthorized\' /var/log/openshift/broker/development.log`
+    else
+      msg_log = `grep \'401 Unauthorized\' /var/log/openshift/broker/production.log`
+    end
+
+    unless msg_log.empty?
+      do_fail <<-CONFLICT
+      Detect an '401 Unauthorized' error on the log. This error indicates the
```

Comment:
Maybe better phrased something like, "This error may indicate inconsistent authentication keys or salts among brokers".